src/io.cr

```diff
@@ -350,12 +350,30 @@ module IO
   # Writes the formatted string to this IO. The format can include
   # format specifiers(subsequences beginning with %). The arguments
   # replace the format specifiers in the format string.
+  # The specifiers that can be supplied to the method are
+  # %s => strings
+  # %b => binary
+  # %o => octal
+  # %d => decimal
+  # %i => decimal
+  # %x => hexadecimal
+  # %X => hexadecimal
+  # %a, %A, %e, %E, %f, %g, %G => char
```

Comment:
I think mentioning the possible input types and giving an example would be good. Also mentioning `%{foo}` and `%<foo>d`. Finally the flags and modifiers for those that take it.